source/release-notes/2.3.0.rst

```diff
@@ -0,0 +1,47 @@
+.. _2.3.0:
+
+===========================================
+Percona Monitoring and Management 2.3.0
+===========================================
+
+:Date: February 17, 2020
+
+`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html>`_ (PMM) is a free and open-source platform for managing and monitoring |MySQL|, |MongoDB|, and |postgresql| performance.
+
+For *PMM* install instructions, see :ref:`pmm.deploying.server` and :ref:`pmm.deploying.client`.
+
+.. note:: PMM 2 is designed to be used as a new installation — your existing
+   PMM 1 environment can't be upgraded to this version.
+
+Improvements and new features
+=============================
+
+* :pmmbug:`5064` and :pmmbug:`5065`: Alert manager available in PMM 2 since this
+  release allows to supply dashboard panels with the rules, which can be
+  evaluated with a specified periodicity to send pre-configured notifications
+  in case if the specified conditions are met.
+* :pmmbug:`5097`: PMM Client docker image is now available for installation in
+  addition to previously available DEB and RPM packages and a tarball.
```

Comment:
This was a part of K8s tasks. please check with @alexanderblinkov-percona and @AlekSi  did we ready to make them public, because I don't see QA  on this issue.
Additionally, https://jira.percona.com/browse/PMM-4796 will include DOC and QA so it's better to announce it after this.

Reply:
ACK. Removed from the current RN

Reply:
That should not be in release notes for sure. We did only the first small step for Percona-Lab, not Percona.